packages/nextra/src/loader.ts

```diff
@@ -321,7 +321,7 @@ const __nextraPageOptions = {
         (themeConfigImport && 'themeConfig: __nextra_themeConfig')
   }
 }
-${finalResult.replace('export default MDXContent;', '')}
+${finalResult.replace(/(export default MDXContent;).*$/, '$1')}
```

Comment:
Nice catch, just one suggestion that we need only replace a last appearance and all code that comes after
<img width="902" alt="image" src="https://github.com/shuding/nextra/assets/7361780/56a462d5-ec51-4e57-a937-95f4b27999cb">
maybe some solution with `String#lastIndexOf` and `String#slice`

could You add also a helpful comment in code why we did it and could you add changeset with `patch`?

Reply:
Thank you for the review.

I apologize for pushing the wrong regex code.
What I intended was 
`${finalResult.replace(/export default MDXContent;(.*)$/, '$1')}`

I used regex to minimize the changes, but I'll think about using `String#lastIndexOf`. 

> could You add also a helpful comment in code why we did it and could you add changeset with patch?

I'll try it later!